share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -28,6 +28,16 @@ macro(rocm_create_package)
         set( CPACK_SET_DESTDIR ON CACHE BOOL "Boolean toggle to make CPack use DESTDIR mechanism when packaging" )
     endif()
 
+    rocm_set_os_id(_os_id)
+    rocm_read_os_release("VERSION_ID" _version_id)
+    
+    #only set CPACK_SYSTEM_NAME for AMD supported OSes
+    if (_os_id_centos OR _os_is_rhel)
+        STRING(CONCAT CPACK_SYSTEM_NAME "el" ${_version_id} ".x86_64")
```

Comment:
`CPACK_SYSTEM_NAME` should be set as a `CACHE` variable, so we can override it if necessary.